src/jira/client/jira-client-deployment-helper.ts

```diff
@@ -16,3 +18,21 @@ export const getDeploymentDebugInfo = (jiraPayload: JiraDeploymentBulkSubmitData
 		commitCount: associations.filter(a => a.associationType === "commit").map(a => a.values?.length || 0).reduce((a, b) => a + b, 0)
 	};
 };
+
+export const extractDeploymentDataForLoggingPurpose = (data: JiraDeploymentBulkSubmitData, logger: Logger): Record<string, any> => {
```

Comment:
Just pulled from JC and placed here so we can import it in. code is same same. Will add a TODO for tests!